src/autogluon/cloud/predictor/cloud_predictor.py

```diff
@@ -60,7 +62,7 @@ class CloudPredictor(ABC):
 
     predictor_file_name = "CloudPredictor.pkl"
 
-    def __init__(self, cloud_output_path, local_output_path=None, verbosity=2):
+    def __init__(self, cloud_output_path: str, local_output_path: Optional[str] = None, verbosity: int = 2) -> None:
```

Comment:
Can we run local mode without passing `cloud_output_path`

Reply:
Well, no. The sagemaker training job always output the artifact to s3